zoo/evaluation/metrics/diversity/diversity_evaluation.py

```diff
@@ -73,6 +74,7 @@ def run_evaluation(self):
             df.set_index(["scenario"], inplace=True)
             for agent in agent_list:
                 for scenario_name, result_data in result_dict.items():
+                    # pytype: enable=attribute-error
```

Comment:
Yes it is because of `result_dict`. `diversity_data_analyze` can return `None` which is what I think is causing this error https://github.com/huawei-noah/SMARTS/blob/a7825623fe32a1a2509d3f7ad97dc61eaaa97ff9/zoo/evaluation/metrics/diversity/diversity_evaluation.py#L44